README.md

```diff
@@ -1 +1,188 @@
 # node-assignment
+
+To reproduce tables in database please use the below statements
```

Comment:
Możesz mieć ustawiony skrypt dodatkowy w npm, który będzie setupował system, lub zawracał info o błędzie jeśli coś poszło nie tak, lub zrobić zmienna środowiskową, która gdy jest podana przy uruchamianiu aplikacji to robi setup środowiska, lub robić przy każdym stacje aplikacji sprawdzenie czy wszystko jest ok, jeśli nie to zrobi co trzeba lub wymyśl swoje rozwiązanie. W README powinno być tylko info o potrzebnych rzeczach do zainstalowania.